source/common/quic/BUILD

```diff
@@ -1,13 +1,12 @@
 load(
     "//bazel:envoy_build_system.bzl",
-    "envoy_cc_extension",
     "envoy_cc_library",
-    "envoy_extension_package",
+    "envoy_package",
```

Comment:
Hmmmm.  I'm inclined to either keep the default visibility of these libraries limited, or comment heavily which ones need to be bazel selected.  We can arguably trust the compile options build to catch us if you prefer that, but at least we should sprinkle some warnings at the top of the file that generally these should be selected with envoy_select_enable_http3 and add per library comments for the ones which are safe to straight up include.
I'm fine with the latter being a follow-up TODO.

Reply:
Yeah I agreed I will add some TODOs.